pkg/corerp/frontend/controller/applications/updatefilter.go

```diff
@@ -70,9 +70,9 @@ func CreateAppScopedNamespace(ctx context.Context, newResource, oldResource *dat
 		}
 
 		namespace := fmt.Sprintf("%s-%s", envNamespace, serviceCtx.ResourceID.Name())
-		if !kubernetes.IsValidObjectName(namespace) {
-			return rest.NewBadRequestResponse(fmt.Sprintf("Application namespace '%s' could not be created: the combination of application and environment names is too long.",
-				namespace)), nil
+		err = kubernetes.ValidateObjectName(namespace)
+		if err != nil {
+			return rest.NewBadRequestResponse(fmt.Sprintf("Application namespace '%s' is invalid: %s", namespace, err.Error())), nil
```

Comment:
just to confirm, this would return an error message similar to the example you provided in the issue?

> "Application namespace 'default-App' is invalid: name default-App is invalid: a lowercase RFC 1123 label must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name',  or '123-abc', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?')"

Reply:
yes, thats the message string we get with this change.